tests/fixtures/invalid_check.yml

```diff
@@ -0,0 +1,30 @@
+name: Corosync configuration file
```

Comment:
would you mind adding a commented line for the omitted id? I'd be clearer why this check is invalid